build-patched-kao-kcmo-images.sh

```diff
@@ -139,13 +139,11 @@ function update_base_image() {
     rhpkg clone containers/${brew_repo}
     pushd ${brew_repo}
     git checkout --track origin/crc-1-rhel-8
-    git checkout --track origin/private-cfergeau
-    git reset --hard origin/crc-1-rhel-8
     sed -i "s!^FROM openshift/ose-base.*!FROM $base_image!" Dockerfile
     git add Dockerfile
     git commit -m "Use OpenShift ${openshift_version} base image"
-    git push origin -f HEAD:private-cfergeau
-    rhpkg container-build --scratch --target crc-1-rhel-8-candidate
+    git push origin
+    rhpkg container-build --target crc-1-rhel-8-candidate
```

Comment:
I'm unconvinced it's usefulu to explicitly specify the target, as it's a unusual way of running `rhpkg container-build`

Reply:
explicit is always better than implicit.

Reply:
I disagree. "oh, they specify --target, they must not be using the default target for that branch. That's odd, what is the default target? And why do they need a different one?"

Reply:
Or they are not using any target so what is the default one? both time user need to look for the doc. As new user of brew I don't know the branch is default target because in case of patch container images we are building tagged images from the branches and if we don't specify the target it would fail so I still prefer to have it explicit.

Reply:
But it's not important to know what target is being used, unless you are debugging that very code. 99% of the code, `rhpkg container-build` is good enough.

Reply:
I understand for folks who are familiar with brew/koji have no issue to know that default target is taken as your branch if not provided but for anyone else who are new to it will going to have bit hard time if he is doing something in different branch and build fails because it will as for target and he have to figure out what could be target.

Anyway I have remove it  but I still think explicit is always better (no harm to put extra info even it is default),  but don't want to block the PR because of this.

Reply:
I'm not going to block this either if you keep the --target, but too much information really is harmful in itself